src/containers/shared/components/DomainLink.tsx

```diff
@@ -7,14 +7,27 @@ export interface Props {
   domain: string
 }
 
+// Matches a protocol (e.g. 'http://' or 'https://') at the start of a string.
+const protocolRegex = /^([a-z][a-z0-9+\-.]*):\/\//
```

Comment:
```suggestion
const PROTOCOL_REGEX = /^([a-z][a-z0-9+\-.]*):\/\//
```
Since it's a global constant

Reply:
Implemented it in the new commit _Refactor: Improved code readability and consistency_ (https://github.com/ripple/explorer/pull/856/commits/53132ca2cc21119414b489a8fd65fc175e3c46ba)